docker/Dockerfile_stack

```diff
@@ -253,7 +250,7 @@ RUN ["/bin/bash","-c","export KOKKOS_TOOLS_HASH=7ee5bcb80a8329c8493da735bc7bad5a
                                \"space-time-stack\" \
                               ) && \
     export KOKKOS_TOOLS_URL=https://github.com/kokkos/kokkos-tools/archive/${KOKKOS_TOOLS_HASH}.tar.gz && \
-    export KOKKOS_TOOLS_ARCHIVE=${PREFIX}/archive/kokkos-tools-${KOKKOS_TOOLS_HASH}.tar.xz && \
+    export KOKKOS_TOOLS_ARCHIVE=${PREFIX}/archive/kokkos-tools-${KOKKOS_TOOLS_HASH}.tar.gz && \
```

Comment:
Is `xz` not available? If it is, why the change?

Reply:
GitHub let you download either a tarball (.tar.gz) or a zipball (.zip) archive.  See [here](https://developer.github.com/v3/repos/contents/#get-archive-link).  We were just changing the extension name by mistake.

Reply:
didnt know there was an xz

Reply:
It does exist.

Reply:
Not on GH though.